src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,62 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private File file;
+
     public void getStatistic(String fromFileName, String toFileName) {
+        writeDataToFile(readAndSort(fromFileName), toFileName);
+
+    }
+
+    public int [] readAndSort(String fromFileName) {
+        file = new File(fromFileName);
+        String [] temp = new String[2];
+        int [] result = new int[3];
+
+        try {
+            BufferedReader reader = new BufferedReader(new FileReader(file));
+            String splitInput = reader.readLine();
+
+            while (splitInput != null) {
+                temp = splitInput.split(",");
+                System.out.println(splitInput);
```

Comment:
Remove the `System.out.println` statement as it is not necessary for the method's functionality and can clutter the output.